server/utils/constants.ts

```diff
@@ -40,3 +40,7 @@ export enum CLUSTER {
 export enum INDEX {
   OPENDISTRO_ISM_CONFIG = ".opendistro-ism-config",
 }
+
+export enum Setting {
+  RolloverAlias = "opendistro.index_state_management.rollover_alias",
```

Comment:
Convert this to Capital snake case.

Reply:
I have a mix of the two at this point. I see on TypeScript docs enums represented this way, whereas I was using capital snake case before just because I was used to using `const SOME_CONSTANT = ...`
Do you have some resources pointing towards using enums/values as capital snake case still?

Reply:
Talked offline, outcome is to stick to one convention. I started writing my enums as habit from JavaScript and noticed TypeScript docs wrote them this way partway through. Planning to migrate other enums that are capital snake case to this format too, just not part of this PR.